core/eolearn/core/core_tasks.py

```diff
@@ -555,6 +557,29 @@ def map_method(self, feature):
         return feature[..., self.bands]
 
 
+class ExplodeBandsTask(EOTask):
+    """Explode a subset of bands from one feature to multiple new features."""
+
+    def __init__(
+        self,
+        input_feature: Tuple[FeatureType, str],
+        output_mapping: Dict[Tuple[FeatureType, str], Iterable[int]],
```

Comment:
as mentioned before, it would be nice to have the options of `Dict[Tuple[FeatureType, str], Union[int, Iterable[int]]]`, so that one can do `{"B04": 3, "B03": 2}` without the pesky brackets